### PR TITLE
Add Unit Test:describe configmap

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
Add Unit Test:describe configmap

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by cblecker** [comment]
@k8s-bot ok to test
/release-note-none

**Comment by cblecker** [comment]
@k8s-bot kops aws e2e test this

File: pkg/printers/internalversion/describe_test.go [hunk 1]
```diff
@@ -111,6 +111,24 @@ func TestDescribeNamespace(t *testing.T) {
 	}
 }
 
+func TestDescribeConfigMap(t *testing.T) {
+	fake := fake.NewSimpleClientset(&api.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "mycm",
+			Namespace: "foo",
+		},
```
Comment (rootfs) [inline pkg/printers/internalversion/describe_test.go]: The test doesn't really test config map, provide a configmap spec here.

Comment (zhangxiaoyu-zidif) [inline pkg/printers/internalversion/describe_test.go]: @rootfs Thanks for your review. I will add spec later. Thanks =)

Comment (zhangxiaoyu-zidif) [inline pkg/printers/internalversion/describe_test.go]: @rootfs I add configmap's Data, PTAL. Thanks very much.

**Comment by rootfs** [comment]
/lgtm

thanks for contributing

**Comment by zhangxiaoyu-zidif** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by zhangxiaoyu-zidif** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by zhangxiaoyu-zidif** [comment]
ping @brendandburns

**Pull request merged**